docs-app/app/components/ui/form.hbs

```diff
@@ -4,7 +4,7 @@
     (concat this.formId "-instructions")
   }}
   aria-labelledby={{if @title (concat this.formId "-title")}}
-  data-test-form={{or @title ""}}
+  data-test-form={{if @title @title ""}}
```

Comment:
The `{{or}}` helper from `ember-truth-helpers` can be misused to perform the ternary operator `??` from JavaScript.

Since `{{strict-or}}` means to perform the or operator `||`, we must use `{{if}}` statement to more clearly state our intent.

(Maybe Ember could provide a helper that corresponds to `??`.)